src/App.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import "./App.css";
+
+function App() {
+  let image;
+  let response = "";
+
+  // When user selects an image
+  function handleImageOnChange(e) {
+    e.preventDefault();
+    image = e.target.files[0];
+  }
+
+  // When user clicks 'Submit' button
+  function uploadImage() {
```

Comment:
Ahh I see thanks for the tip, sorry if it wasn't clear but I was thinking making a backend would effectively double the work having the same image passed to the backend and then to the azure api, but considering for security and the exposed key it would be a good idea to build a backend for this lol